pkg/scheduler/scheduler_test.go

```diff
@@ -172,6 +174,12 @@ func TestScheduler(t *testing.T) {
 			sendPod:     deletingPod("foo"),
 			algo:        mockScheduler{"", nil},
 			eventReason: "FailedScheduling",
+		}, {
+			sendPod:          podWithID(testBindingPodName, ""),
+			algo:             mockScheduler{testNode.Name, nil},
+			expectBind:       testBinding, // testBinding has a name different than testBindingPodName
```

Comment:
I am confused. Why is the name different?

Reply:
This is confusing and not that useful. I already added full tests in `TestGetBinderFunc` in `pkg/scheduler/factory/factory_test.go` so this should have been removed.